model/client4.go

```diff
@@ -8078,3 +8082,21 @@ func (c *Client4) GetAppliedSchemaMigrations() ([]AppliedMigration, *Response, e
 	}
 	return list, BuildResponse(r), nil
 }
+
+// Usage Section
+
+// GetPostsUsage returns rounded off total usage of posts for the instance
+func (c *Client4) GetPostsUsage() (*PostsUsage, *Response, error) {
+	r, err := c.DoAPIGet(c.usageRoute()+"/posts", "")
+	if err != nil {
+		return nil, BuildResponse(r), err
+	}
+	defer closeBody(r)
+
+	var usage *PostsUsage
+	if err = json.NewDecoder(r.Body).Decode(&usage); err != nil {
+		return nil, BuildResponse(r), err
+	}
+
+	return usage, BuildResponse(r), nil
+}
```

Comment:
Non-blocker: This can just be: 

```suggestion
	err = json.NewDecoder(r.Body).Decode(&usage)
	return usage, BuildResponse(r), err
}
```